dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -262,7 +262,7 @@ public class ESContentletAPIImpl implements ContentletAPI {
 
     public static final String UNIQUE_PER_SITE_FIELD_VARIABLE_NAME = "uniquePerSite";
 
-
+    private final String IS_COPY = "isCopy";
```

Comment:
Looks like we already have a content property https://github.com/dotCMS/core/blob/4c3a898a308cadc2247a06fbee5397ed629d04f8/dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java#L170 let's check if we can use it. If not let's add the property to the class mentioned, so you can call it from diff places.